File: app/src/main/java/com/fastaccess/provider/timeline/handler/StrikethroughHandler.java
package com.fastaccess.provider.timeline.handler;

import android.text.SpannableStringBuilder;
import android.text.style.StrikethroughSpan;

import net.nightwhistler.htmlspanner.TagNodeHandler;

import org.htmlcleaner.TagNode;

public class StrikethroughHandler extends TagNodeHandler {

    @Override public void handleTagNode(TagNode node, SpannableStringBuilder builder, int start, int end) {
        builder.setSpan(new StrikethroughSpan(), start, end, 33);
    }

}

File: markdown/src/main/java/com/fastaccess/markdown/spans/StrikethroughHandler.kt
package com.fastaccess.markdown.spans

import android.text.SpannableStringBuilder
import android.text.style.StrikethroughSpan
import net.nightwhistler.htmlspanner.SpanStack
import net.nightwhistler.htmlspanner.TagNodeHandler
import org.htmlcleaner.TagNode

class StrikethroughHandler : TagNodeHandler() {

    override fun handleTagNode(node: TagNode?, builder: SpannableStringBuilder?, start: Int, end: Int, spanStack: SpanStack?) {
        builder?.setSpan(StrikethroughSpan(), start, end, 33)
    }
}